datanode/gateway/graphql/schema.graphql

```diff
@@ -3061,6 +3061,9 @@ enum Side {
 
 "The interval for trade candles when subscribing via Vega GraphQL, default is I15M"
 enum Interval {
+  "block interval"
```

Comment:
How can someone define this, is it by block height, number of blocks, or something else? It would be good to specify in the docs?

Reply:
It's the interval which corresponds to a single block in the chain, e.g. for trades candles it is all the trades in a block.. so it's not precisely a fixed interval, it depends on how frequently they are generated... I'll improve the docs to make that clear